tests/CMakeLists.txt

```diff
@@ -178,7 +178,7 @@ if(BUILD_GENIVI)
                      src/logger.cc
                      tests/rvisotaclient_test.cc)
     add_executable(aktualizr_test_rvi_client ${TEST_RVI_SRC})
-    target_link_libraries(aktualizr_test_rvi_client rvi ${TEST_LIBS} gtest)
+    target_link_libraries(aktualizr_test_rvi_client ${TEST_LIBS} gtest)
```

Comment:
No it is not, it is linker flag, in tests I don't use rvi_lib, instead I defined all symbols in test. But rvi.h is required so the problem is with compilation in finding include files.